physics/GWD/ugwpv1_gsldrag.F90

```diff
@@ -44,7 +44,7 @@ module ugwpv1_gsldrag
     use cires_ugwpv1_solv2,    only:  cires_ugwpv1_ngw_solv2
     use cires_ugwpv1_oro,      only:  orogw_v1
 
-    use drag_suite,            only:  drag_suite_run
+    use drag_suite,            only:  drag_suite_run, drag_suite_psl
```

Comment:
I see what happened now. This is a bad example to follow.

Reply:
Agree, the drag_suite_psl should be called inside the drag_suite_run